Misc/Positions_Area Positions 2 External.txt

```diff
@@ -36,22 +39,17 @@ EISN_N_CTR:Shannon Control:122.970:SNN:SNN:EISN:CTR:EISN:CTR:6600:6667
 EISN_S_CTR:Shannon Control:135.600:SNS:SNS:EISN:CTR:EISN:CTR:6600:6667
 EISN_W_CTR:Shannon Control:125.870:SNW:SNW:EISN:CTR:EISN:CTR:6600:6667
 EISN_I_CTR:Shannon Information:134.270:SNI:I:EISN:CTR:EISN:CTR:6600:6677
-EIDW_N_CTR:Dublin Control:129.170:DN:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:124.650:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:126.250:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:120.050:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_H_CTR:Dublin Control:119.550:DH:D:EIDW:CTR:EIDW:CTR::
-EIDW_N_APP:Dublin Control:129.170:DN:D:EIDW:APP:EIDW:CTR::
```

Comment:
Are these not APP positions we have defined too? What has become of these?

Reply:
Primarily callsign changes and new positions added, no longer a simple north/south split, as it's now North High (1), North Low (2), South High (3), and South Low (4). (don't quote me on those numbers, i haven't controlled there in a while)

Reply:
OK, but they are all CTR positions? And will never log on with the APP suffix? Notice how we defined the same frequencies twice just with the APP suffix to ensure the controller would always be identified?